java/common/rules/java_runtime.bzl

```diff
@@ -152,6 +152,7 @@ def _java_runtime_rule_impl(ctx):
         platform_common.TemplateVariableInfo({
             "JAVA": java_binary_exec_path,
             "JAVABASE": java_home,
+            "JAVABASE_RUNFILES": java_home_runfiles_path,
```

Comment:
Not adding an equivalent for the `java` binary makes this difficult to use cross-platform (think Windows).

I don't think the deprecated env var `JAVA_RUNFILES` would necessarily conflict with a make var. It's something we could likely just get rid of at this point anyway.

But to avoid confusion, how about calling this `JAVA_ROOTPATH` or `JAVABASE_ROOTPATH` to match `$(rootpath ...)`?

Reply:
Sure, I've tried to address this.